ERROR: unable to add snapshot: snapshot with name snap1 already exists
